python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1639,11 +1665,11 @@ def test___setstate__():
         ray_df.__setstate__(None)
 
 
-def test___delitem__():
-    ray_df = create_test_dataframe()
-
-    with pytest.raises(NotImplementedError):
-        ray_df.__delitem__(None)
+@pytest.fixture
+def test___delitem__(ray_df, pd_df):
```

Comment:
I think we should probably copy these dataframes so we don't affect other tests. Deleting here will delete in-place and change for all future tests, I believe.

Reply:
Fixed!